### PR TITLE
GG-5646

### PR DESCRIPTION
**Pull request by martin-armstrong**

### CODE REVIEW

File: app/uk/gov/hmrc/emailverification/services/AuditService.scala [hunk 1]
```diff
@@ -249,6 +249,38 @@ class AuditService @Inject() (
...
+      "origin" -> verifyEmailRequest.origin,
+      "continueUrl" -> verifyEmailRequest.continueUrl,
+      "deskproServiceName" -> verifyEmailRequest.deskproServiceName.getOrElse("-"),
+      "accessibilityStatementUrl" -> verifyEmailRequest.accessibilityStatementUrl,
+      "pageTitle" -> verifyEmailRequest.pageTitle.getOrElse("-"),
+      "backUrl" -> verifyEmailRequest.backUrl.getOrElse("-"),
+      "emailAddress" -> verifyEmailRequest.email.fold("-")(_.address),
+      "emailEntryUrl" -> verifyEmailRequest.email.fold("-")(_.enterUrl),
+      "lang" -> verifyEmailRequest.lang.fold("-")(_.value),
+      "status" -> responseStatus.toString
```
Comment (altunhasan) [inline app/uk/gov/hmrc/emailverification/services/AuditService.scala]: according to GG-5646 : "statusCode"

File: app/uk/gov/hmrc/emailverification/services/AuditService.scala [hunk 2]
```diff
@@ -249,6 +249,38 @@ class AuditService @Inject() (
...
+    sendEvent("VerifyEmailRequest", details, "HMRC Gateway - Email Verification - a VerifyEmailRequest has been made to verify an email address belongs to the requestor.")
+  }
+
+  def sendEmailVerificationOutcomeRequestEvent(credId: String, emailsStatusJson: String, responseStatus: Int)(implicit request: Request[_]): Future[Unit] = {
+    val details = Map(
+      "credId" -> credId,
+      "bearerToken" -> hc.authorization.getOrElse(Authorization("-")).value,
+      "emails" -> emailsStatusJson,
+      "status" -> responseStatus.toString
+    )
```
Comment (altunhasan) [inline app/uk/gov/hmrc/emailverification/services/AuditService.scala:282]: according to GG-5646  userAgentString and statusCode do not match

**Review by altunhasan — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/emailverification/services/AuditService.scala (hunk 1 above)
Comment (martin-armstrong) [reply app/uk/gov/hmrc/emailverification/services/AuditService.scala]: thanks

File: app/uk/gov/hmrc/emailverification/services/AuditService.scala (hunk 2 above)
Comment (martin-armstrong) [reply app/uk/gov/hmrc/emailverification/services/AuditService.scala:282]: thanks

**Review by altunhasan — APPROVED** [review]

**Pull request merged**